### ISSUE TITLE
(chrisying): fused batch norm is currently not supported for 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#3738]
multi-axis batch norm and by extension virtual batches. In some cases,
it might be possible to use fused batch norm but would require reshaping
the Tensor to 4D with the axis in 1 or 3 (preferred 1) which is
particularly tricky. A compromise might be to just support the most
common use case (turning 5D w/ virtual batch to NCHW)

----
*opened via [imdone.io](https://imdone.io) from a code comment on [693b339ab2f062ec5bbb29f976c5d1fd94fbffa5](https://github.com/imdone/tensorflow/commit/693b339ab2f062ec5bbb29f976c5d1fd94fbffa5) by Francois Chollet fchollet@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/python/keras/_impl/keras/layers/normalization.py#L236-L246
